src/Atm.java

```diff
@@ -0,0 +1,245 @@
+public class Atm {
+    final String pinCode = "1234";
+    private boolean cardInside = false;
+    private int balance = (int) (50 + 100000 * Math.random());
+    private final int FIFTY = 50;
+    private final int ONE_HUNDRED = 100;
+    private final int TWO_HUNDRED = 200;
+    private final int FIFTH_HUNDRED = 500;
+    private final int ONE_THOUSAND = 1000;
+    private final boolean paperPresence = Math.random() < 0.8;
+
+    DataValidation checkInputItem = new DataValidation();
+
+    public int getBalance() {
+        return balance;
+    }
+
+    public void setBalance(int balance) {
+        this.balance = balance;
+    }
+
+    public boolean isCardInside() {
+        return cardInside;
+    }
+
+    public void setCardInside(boolean cardInside) {
+        this.cardInside = cardInside;
+    }
+
+    void drawItemMenu(Menus menus) {
+        switch (menus) {
+            case INPUT_CARD:
+                System.out.format("\tPlease insert your Card!\t\n" +
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.